web/themes/custom/lex/old_scss/_layout.sass

```diff
@@ -168,7 +168,7 @@ main
 
   .path-node.bundle-news_article &
     @media (min-width: $screen-desk-max)
-      max-width: calc(940px + 10%)
+      max-width: calc(1000px + 10%)
```

Comment:
In all reality we'd want this to stay as close to `80ch` as we could. I pulled the 940px to get as close to that as possible since 80ch was pretty tight with the sidebar. Sort of went along the lines of 80ch + sidebar + padding.